app.js

```diff
@@ -1,15 +1,23 @@
-// create an express app
-const express = require("express")
-const app = express()
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const port = 3000;
 
-// use the express-static middleware
-app.use(express.static("public"))
+const { initialiseDbConnection } = require('./db/connect.db.js');
 
-// define the first route
-app.get("/", function (req, res) {
-  res.send("<h1>Hello World!</h1>")
-})
+const productRouter = require('./routes/products.routes.js');
 
-// start the server listening for requests
-app.listen(process.env.PORT || 3000, 
-	() => console.log("Server is running..."));
\ No newline at end of file
+// to open requests from around
+app.use(cors())
+app.use(express.static('public'))
+initialiseDbConnection();
+
+app.use('/products', productRouter);
+
+app.get('/', (req, res) => {
+  res.json({ 'Hello From Farmers Grocery!': 'API' })
+});
+
```

Comment:
Hi Shubham,
Over here could you please add an invalid route handler?
Something like 404 route not found!